main.py

```diff
@@ -1,121 +1,131 @@
-# TOKEN = os.getenv("TOKEN")
-
-
-def breakliner():
-    print(f"{BREAK:-^30}")
-    return
-
-
-def clearScreen():
-    if sys.platform == "windows":
-        os.system("cls")
-    else:
-        os.system("clear")
-    return
-
-
-def exitQuery():
-    exitQ = input("Exit? ")
-    if exitQ == "y":
-        exit()
-    elif exitQ == "n":
-        clearScreen()
-        OWMCITY()
-
-def data_processing():
-    main = data["main"]
-    humidity = main["humidity"]
-    pressure = main["pressure"]
-    temp = main["temp"]
-    temp_min = main["temp_min"]
-    temp_max = main["temp_max"]
-
-    def ktc(temp, temp_min, temp_max):
-        temp = temp - 273.15
-        temp_min = temp_min - 273.15
-        temp_max = temp_max - 273.15
-        print("Temperature: ", temp, "°C")
-        print("Minimum Temperature: ", temp_min, "°C")
-        print("Maximum Temperature: ", temp_max, "°C")
-
-    CITY = data["name"]
-    sys = data["sys"]
-    country = sys["country"]
-    CITY2 = CITY + "," + " " + country
-    if CITY == CITY2:
-        pass
-    else:
-        CITY = data["name"]
-        sys = data["sys"]
-        country = sys["country"]
-        CITY2 = CITY + "," + " " + country
-
-    w_main = data["weather"][0]["main"]
-    w_desc = data["weather"][0]["description"]
-    pressure = main["pressure"]
-    visibility = data["visibility"]
-    visibility_new = visibility / 1000
-    wind = data["wind"]["speed"]
-
-    def printData():
-        breakliner()
-        print(f"{CITY2:-^30}")
-        print(f"Weather: {w_main}:- {w_desc}")
-        ktc(temp, temp_min, temp_max)
-        print(f"Humidity: {humidity}%")
-        print(f"Pressure: {pressure} hPa")
-        print(f"Wind speed: {wind} m/s")
-        print(f"Visibility: {visibility} m (or) {visibility_new} km")
-
-    printData()
-        
-def OWMCITY():
-    # City Name request
-    global CITY
-    CITY = input(
-        "Enter City Name (should be in compliance to OpenWeatherMap's City Index): "
-    )
-    if CITY == "exit":
-        clearScreen()
-        print("Three")
-        sleep(1)
-        clearScreen()
-        print("Two")
-        sleep(1)
-        clearScreen()
-        print("One.")
-        sleep(1)
-        clearScreen()
-        print("Exiting...")
-        sleep(0.5)
-        exit()
-
-    else:
-        pass
-    # The URL in actuality be like:
-    URL = BASE_URL + "q=" + CITY + "&appid=" + TOKEN
-
-    # PROD: Request Response
-    global resp_PROD
-    resp_PROD = requests.get(URL)
-    if resp_PROD.status_code == 200:
-        pass
-    else:
-        raise TypeError("Oops, wrong city name or code?")
-
-    global data
-    data = resp_PROD.json()
-    
-    data_processing()
-    exitQuery()
-
-OWMCITY()
+from __future__ import annotations
+
+from platform import system
+from tkinter import Menu, Tk, messagebox
+from tkinter.ttk import Button, Entry, Frame, Label, Style
+
+from requests import Response, get
+
+
+def self_return_decorator(func, *args, **kwargs):
```

Comment:
I'm not sure whether a fluent API has any use case here.
`App().about().OWMCITY().resize_app()` ?

Reply:
Not necessary, no.